docs/usermanual/index.md

```diff
@@ -175,3 +175,154 @@ You're now ready to go! Log in your AWS account, find the instance (you can sear
+
+```text
+{
+  # Access credentials to Account A
+  'src_profile': {
+      'subscriptionId': xxx,
+      'tenantId': xxx,
+      'clientId': xxx,
+      'clientSecret': xxx
+  },
+  # Access credentials to Account B
+  'dst_profile': {
+      'subscriptionId': yyy,
+      'tenantId': yyy,
+      'clientId': yyy,
+      'clientSecret': yyy
+  },
+  ...
+}
+```
+
+Alternatively, if you are working on only one account and do not require cross-account disk copies, you can define the 
+following environment variables:
+
+```text
+$: export AZURE_SUBSCRIPTION_ID=xxx
+$: export AZURE_CLIENT_ID=xxx
+$: export AZURE_CLIENT_SECRET=xxx
+$: export AZURE_TENANT_ID=xxx
+```
+
+### Disk snapshot
+
+#### Scenario 1
+
+You have a disk `disk1` in an Azure account, in `eastus` which you would like to make a copy 
+of in the same account and in the same region.
+
+#### Scenario 2
+
+You have a disk `disk1` in an Azure account, in `eastus` which you would like to make a copy 
+of in the same account but in a different region `westus`.
+
+#### Scenario 3
+
+You have a disk `disk1` in an Azure account (Account A) which you would like to make a copy 
+of in a different account (Account B).
+
+#### Using the library and the CLI
+
+To make the copy, import the `forensics` package and use the `CreateDiskCopy` method.
+
+```python
+from libcloudforensics.providers.azure import forensics
+
+# Scenario 1. Assumes credentials are configured through environment variables
+copy = forensics.CreateDiskCopy('resource_group_name',
+                                disk_name='disk1')
+      
+# Scenario 2. Assumes credentials are configured through environment variables                          
+copy = forensics.CreateDiskCopy('resource_group_name',
+                                disk_name='disk1',
+                                region='westus')
+                                
+# Scenario 3. Assumes credentials are configured through profiles in ~/.azure/credentials.json
+copy = forensics.CreateDiskCopy('resource_group_name',
+                                disk_name='disk1',
+                                src_profile='src_profile',
+                                dst_profile='dst_profile')
+```
+
+The equivalent CLI command is:
+
+```bash
+# Scenario 1. Assumes credentials are configured through environment variables
+cloudforensics az 'resource_group_name' copydisk --disk_name='disk1'
+
+# Scenario 2. Assumes credentials are configured through environment variables 
+cloudforensics az 'resource_group_name' copydisk --disk_name='disk1' --region='westus'
+
+# Scenario 3. Assumes credentials are configured through profiles in ~/.azure/credentials.json
+cloudforensics az 'resource_group_name' copydisk --disk_name='disk1' --src_profile='src_profile' --dst_profile='dst_profile' 
+```
+
+#### Cross-account / region sharing
+
+Sharing resources between regions / account is not as straightforward in Azure as in GCP, and extra steps must be taken.
```

Comment:
This isn't a very useful sentence, especially if someone is working purely with Azure and is searching the documentation directly on how to use this project.

Reply:
Ah yeah; removed